docs/UI/CreateWindow/index.md

```diff
@@ -0,0 +1,106 @@
+## Create window
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Create a new window on the display with the specified window type.
+
+In order to work with widget windows, the app needs to be able to create them. One app can create one or multiple widgets.
+
+This request is sent by SDL to HMI to create one or multiple widget windows.
+
+!!! must
+  1. Create and store a list of available widgets 
+  2. Notify SDL about widget activation/deactivation
+  3. Reject bidirectional window content duplication 
```

Comment:
Can you please elaborate on number 3?

Reply:
Should I rephrase p.3 into smth `Reject RPCs sent to already created windows`?

Reply:
@JackLivio added some description to p.3 in 7f8e199442f678cd5746d382011502684d7f5c02